### PR TITLE
[GETUNAME] Improve Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Lines 11001-44025.

Addendum to 0887c1c (#5323).

### CODE REVIEW

**Review by GeoB99 — APPROVED** [review]

**Review by s0mt0chukwu — APPROVED** [review]

File: dll/win32/getuname/lang/ro-RO.rc [hunk 1]
```diff
@@ -9833,4979 +9833,4982 @@ BEGIN
...
-    44019 "Meetei Mayek Digit Three"
-    44020 "Meetei Mayek Digit Four"
-    44021 "Meetei Mayek Digit Five"
-    44022 "Meetei Mayek Digit Six"
-    44023 "Meetei Mayek Digit Seven"
-    44024 "Meetei Mayek Digit Eight"
-    44025 "Meetei Mayek Digit Nine"
+    11001 "Linie înclinată dublă mai mic decât or egal cu"
+    11002 "Linie înclinată dublă mai mare decât sau egal cu"
+    11003 "Relation binară de solidus triplu"
```
Comment (reactos-contributor) [inline dll/win32/getuname/lang/ro-RO.rc:9838]: This looks only partially translated.

File: dll/win32/getuname/lang/ro-RO.rc [hunk 2]
```diff
@@ -9833,4979 +9833,4982 @@ BEGIN
...
+    11088 "Steaua medie albă"
+    11089 "Steaua mică neagră"
+    11090 "Steaua mică albă"
+    11091 "Pentagon negru spre dreapta"
+    11092 "Pentagon alb spre dreapta"
+    11093 "Cerc mare și gros"
+    11094 "Oval gros cu oval înăuntru"
+    11095 "Cerc gros cu cerc înăuntru"
+    11096 "Cerc gros"
+    11097 "Crucea Sfântului Andrei puternic înconjurată"
```
Comment (reactos-contributor) [inline dll/win32/getuname/lang/ro-RO.rc:9932]: Please revise your "puternic înconjurată" chosen syntagm.

File: dll/win32/getuname/lang/ro-RO.rc [hunk 3]
```diff
@@ -9833,4979 +9833,4982 @@ BEGIN
...
+    11185 "Săgeată în formă de panglică în jos și spre dreapta"
+    11186 "Săgeată în formă de panglică în sus și spre stânga"
+    11187 "Săgeată în formă de panglică în sus și spre dreapta"
+    11188 "Săgeată în formă de panglică spre stânga sus"
+    11189 "Săgeată în formă de panglică spre dreapta sus"
+    11190 "Săgeată în formă de panglică spre stânga jos"
+    11191 "Săgeată în formă de panglică spre dreapta jos"
+    11192 "Săgeată albă în sus din bară cu bară orizontală"
+    11193 "Vârf de săgeată în sus într-o cutie dreptunghiulară"
+    11197 "Urnă cu x moale"
```
Comment (reactos-contributor) [inline dll/win32/getuname/lang/ro-RO.rc:10025]: Please remember how you've chosen to translate "light" & "heavy" and be consistent.

File: dll/win32/getuname/lang/ro-RO.rc [hunk 4]
```diff
@@ -9833,4979 +9833,4982 @@ BEGIN
...
+    11198 "X încercuit"
+    11199 "X încercuit și greu"
```
Comment (reactos-contributor) [inline dll/win32/getuname/lang/ro-RO.rc:10027]: Please pay attention to your chosen translation for "bold" (i.e. avoid overlap with other translated terms) and be consistent.

**Review by reactos-contributor — CHANGES_REQUESTED** [review]
Here are a few findings. I'll most likely come back soon with more.

File: dll/win32/getuname/lang/ro-RO.rc (hunk 1 above)
Comment (AndreiMiloiu) [reply dll/win32/getuname/lang/ro-RO.rc:9838]: 44025 is already translated. It is the last translated line from this PR.

**Review by reactos-contributor — APPROVED** [review]
Although there still are some small things that I'd prefer changed, like to capitalize CJK (for being an acronym), or that I'd have "Pătratul" instead of "Pătratul lui" (resource № 13056 and after), I nevertheless approve the PR.

**Comment by AndreiMiloiu** [comment]
Sorry for CJK capitalization. I think that I accidentaly reverted (or re-reverted) it.

**Comment by AndreiMiloiu** [comment]
And also, sorry for OCR capitalization. It was the same mistake I made as for CJK capitalization.

**Pull request merged**